<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchUser</class>
 <widget class="QMainWindow" name="SearchUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BÃºsqueda de usuario</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="line_edit_search">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>391</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Usuarios:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="list_widget_users">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>391</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_widget_users">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>391</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
